src/binarySearch.js

```diff
@@ -0,0 +1,28 @@
+export default function binarySearch( arr, query){
+  if( arr == null ){
+    return 'Cannot search null'
```

Comment:
these should probably throw errors instead of return this error message as a string